adcm_client/base.py

```diff
@@ -69,6 +73,28 @@ class PagingEnds(Exception):
     """There are no more data in paginated mode."""
 
 
+class IncompatibleVersion(Exception):
+    """Incompatible version, upgrade version ADCM."""
+    def __init__(self, method_name='', version='', message=None):
+        if message is None:
+            self.message = (f'The "{method_name}" method works with versions older {version},'
+                            f' upgrade version ADCM.')
+        else:
+            self.message = message
+        super(IncompatibleVersion, self).__init__(self.message)
+
+
+def version_compatibility_check(version):
```

Comment:
I don't like this name. We are not checking version. We a checking  minimal supported version. So it should be minimal or min in name of function. Something like "@min_server_version("2020.01.20.20")"

Reply:
Ok